project/database/migrations/2019_11_26_133751_create_posts_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreatePostsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+	public function up() {
+		Schema::create( 'posts', function ( Blueprint $table ) {
+			$table->bigIncrements( 'id' );
```

Comment:
`bigIncrements` were added in Laravel 5.8 as default increment field, but I think it's better to use `increments` as it will reduce size of id fields and the sizes of foreign keys for such tables.